// node-dictionary/test/mock/query.js
module.exports = {
  tested: {
    word: 'go',
    explains: {
      translation: ['去'],
      basic: {
        'us-phonetic': 'go',
        phonetic: 'gəʊ',
        'uk-phonetic': 'gəʊ',
        explains: [
          'n. 去；进行；尝试',
          'vi. 走；达到；运转；趋于',
          'vt. 忍受；出产；以…打赌',
          '[复数 goes 第三人称单数 goes 过去式 went 过去分词 gone 现在分词 going]'
        ]
      },
      query: 'go',
      errorCode: 0,
      web: [{
        value: ['围棋', '走', '去'],
        key: 'Go'
      }, {
        value: ['开始', '前进', '先走'],
        key: 'go ahead'
      }, {
        value: ['上升', '上涨', '增长'],
        key: 'go up'
      }]
    }
  },
  expected: {
    url: 'http://fanyi.youdao.com/openapi.do?keyfrom=node-dictionary&key=110082369&type=data&doctype=json&version=1.1&only=dict&q=go',
    result: ['  \n', '  go\n', '  \n', '  英 [gəʊ]  美 [go]\n\n', '    n. 去；进行；尝试\n', '    vi. 走；达到；运转；趋于\n', '    vt. 忍受；出产；以…打赌\n', '    [复数 goes 第三人称单数 goes 过去式 went 过去分词 gone 现在分词 going]\n', '  \n', '  网络释义\n\n', '    Go  围棋,走,去\n', '    go ahead  开始,前进,先走\n', '    go up  上升,上涨,增长\n', '  \n', '  Powered by Youdao\n']
  }
}
